File: 01272023/9_69_Food_Classifying/java/FoodClassifying.java
import java.util.ArrayList;
import java.util.List;
import java.util.Random;

public class FoodClassifying {
    public static void main(String[] args) {
        List<String> foodList = new ArrayList<>();
        foodList.add("apple");
        foodList.add("grape");
        foodList.add("bacon");
        foodList.add("steak");
        foodList.add("worm");
        foodList.add("dirt");

        Random randomizer = new Random();

        String food = foodList.get(randomizer.nextInt(foodList.size()));
        System.out.println(food);
    
        if (food == "apple" || food == "grape") {
            System.out.println("fruit");
        } else if (food == "bacon" || food == "steak") {
            System.out.println("meat");
        } else if (food == "dirt" || food == "worm") {
            System.out.println("yuck");
        }
    }
}

File: 01272023/09_69_Food_Classifying/python/food_classifying.py
from random import choice


food: list = choice(['apple','grape', 'bacon', 'steak', 'worm', 'dirt'])

if food == 'apple' or food == 'grape':
    print('fruit')
elif food == 'bacon' or food == 'steak':
    print('meat')
elif food == 'dirt' or food == 'worm':
    print('yuck')